### DISCUSSION TITLE
Lightbulb affects only one of two monitors. How to make it work with both or at least choose the one affected?

### DISCUSSION THREAD

**Discussion by astuteficus** [Tyrrrz/LightBulb#330] [Q&A]
I have two monitors and Lightbulb was working with the main one. The second was not affected. It was not very convenient but ok. After an update, the second monitor gets adjusted, but not the first one. 
How to make it work with both or at least choose the one affected?
Thanks for a great app!

**Comment by Tyrrrz** [comment]
Are they connected using different ports or to different adapters (e.g. one to GPU, one to Motherboard)?

**Comment by astuteficus** [comment]
I have ASUS ZenBook Duo. They are both internal monitors of a laptop. The laptop also has two adapters (motherboard Intel + Nvidia). 😅

**Reply by Tyrrrz** [reply]
Do you know if they are internally wired to different adapters or the same one?

**Comment by astuteficus** [comment]
Windows says they are both connected to Intel adapter

**Reply by Tyrrrz** [reply]
Weird. Does turning on gamma polling in LightBulb settings help at all?

**Reply by astuteficus** [reply]
Unfortunately, turning it on didn't help. Maybe there is a setting in the register that sets the active monitor?

**Reply by Tyrrrz** [reply]
I'm not aware of one. The only time something similar was reported before, it was with monitors connected to different adapters (although in that case the gamma on one of them never updated, not just sometimes). So unfortunately I don't know what is causing the issue.
